libs/structwalk/walktype_test.go

```diff
@@ -27,11 +32,7 @@ func TestTypeNilCallback(t *testing.T) {
 }
 
 func TestTypeNil(t *testing.T) {
-	assert.Equal(t, map[string]any{}, getFields(t, reflect.TypeOf(nil)))
-}
-
-func TestTypeScalar(t *testing.T) {
```

Comment:
Removed this test because it does not have useful semantics. WalkType only makes sense on structs (or map / slice of structs). My changes break this test because here 5 is the root even though it's scalar.

Reply:
Reverted.